scribblings/api.scrbl

```diff
@@ -41,37 +27,46 @@ not wrapped by a @racket[chaperone?] or @racket[impersonator?], nor may its elem
 coverage information is still collected when test fail.  Test coverage info is added to existing
 coverage info.}
 
-@defproc[(clear-coverage! [environment environment? (current-coverage-environment)]) any]{
+@defproc[(clear-coverage! [environment environment? (current-cover-environment)]) any]{
 Clears all coverage information.}
 
-@defproc[(get-test-coverage [environment environment? (current-coverage-environment)]) coverage/c]{
+@defproc[(get-test-coverage [environment environment? (current-cover-environment)]) coverage/c]{
 Gets the current coverage information.}
-@defproc[(make-covered? (coverage file-coverage/c) (path path-string?))
-         (->* (exact-positive-integer?)
-              (#:byte? boolean?)
-              (or/c 'covered 'uncovered 'irrelevant))
-         ]{
-Given some location in a file and the
-coverage information for that file @racket[make-covered?] returns
-a functions that determines if some @racket[1] indexed character or byte location
-in that file is covered. By default it checks character locations.
-
-There are three possible results: @itemize[@item{@racket['irrelevant] --- The location is not
-considered relevant to coverage information.  It is either not in the coverage information; is in a
-submodule specified by @racket[irrelevant-submodules]; is a @racket[begin-for-syntax] form; or lexes
-(in the sense of that languages, @racket[_color-lexer]) as a comment or whitespace.}
+
+@defthing[coverage/c
+          contract?
+          #:value (-> any/c exact-positive-integer?
+                       (or/c 'covered 'uncovered 'irrelevant))]{
+Coverage information is represented as a mapping from a file's key and a  character
+location to whether or not that location in the file was covered.
+
+The files key is determined by what @racket[syntax-source] is on the syntax of the program after
+reading it. Typically this is the @racket[string?] for of the absolute path of the file path. If
+coverage was run manually, as in the @tech{Lower Lever API}, this value may be something
+else.
+
+The character locations are @racket[1] indexed.
+
+There are three possible results for coverage: @itemize[@item{@racket['irrelevant] --- The location
+is not considered relevant to coverage information.  It is either not in the coverage information;
+is in a submodule specified by @racket[irrelevant-submodules]; is a @racket[begin-for-syntax] form;
+or lexes (in the sense of that language's @racket[_color-lexer]) as a comment or whitespace.}
```

Comment:
We've had this discussion before. I think it should be the first thing in the section. That's how it was before this change.